tests/ui/closures/closure-return-type-mismatch.rs

```diff
@@ -15,3 +15,8 @@ fn main() {
         b
     };
 }
+
```

Comment:
Couldn't find a better place to put id, I didn't want to create a new file for this.